server/controllers/article.controller.js

```diff
@@ -204,5 +204,25 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * Delete article
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async deleteArticle(req, res) {
+    const {
+      params: { slug }, user: { id }
+    } = req;
+
+    const article = await Article.destroy({
+      where: {
+        slug,
+        userId: id,
+      }
+    });
+    if (article) return Response(res, 200, `Successfully deleted ${article} article`);
```

Comment:
There should a check if the article was not deleted successfully.

Reply:
Alright...will put that

Reply:
Done